src/app/_utils/shuffleArray.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Randomly shuffles the elements of an array and returns a new array using the Fisher-Yates (Durstenfeld) shuffle algorithm.
```

Comment:
Why are we doing this? Could it happen that the user clicks on the card, and on the return to the Ecosystem page, the card is not shown because of the shuffle?

Reply:
Also this happens - clicking on the card navigates to /ecosystem/slug, but the breadcrumbs don't navigate to the same place. 🤔 not sure if it's a big deal, but it would be tricky to find that card again if you don't remember which page you were on.

https://github.com/FilecoinFoundationWeb/filecoin-foundation/assets/50910606/2ddff16d-eb13-44d6-ad5c-ba098a6634a4

Reply:
> Could it happen that the user clicks on the card, and on the return to the page, the card is not shown because of the shuffle?

This is a good point. Need to align with @filipagr and Heath on expectations here. Maybe it would better to have a button that users can click to shuffle them if they want. Or something like that.

Reply:
> Also this happens - clicking on the card navigates to /ecosystem/slug, but the breadcrumbs don't navigate to the same place. 🤔 not sure if it's a big deal, but it would be tricky to find that card again if you don't remember which page you were on.

Not sure I understand. What would be the expected behaviour? This is a section of featured Ecosystem Projects on the homepage. Clicking on a single project brings you to the relevant single Ecosystem project page. Following `/ecosystem` goes to the Ecosystem page.

Reply:
@barbaraperic After raising your concerns, I talked to design and marketing - And we decided that these will be featured projects that the content team can set. New CR reflects this.

Let me know if this is still something you want to discuss issue? [>>](https://github.com/FilecoinFoundationWeb/filecoin-foundation/pull/106#discussion_r1586273635)